cmd/kubelet/app/server.go

```diff
@@ -595,6 +595,12 @@ func run(ctx context.Context, s *options.KubeletServer, kubeDeps *kubelet.Depend
 		// make a separate client for events
 		eventClientConfig := *clientConfig
 		eventClientConfig.QPS = float32(s.EventRecordQPS)
+
+		// If the value of EventRecordQPS is 0, there is no limit enforced
+		if eventClientConfig.QPS == 0.0 {
```

Comment:
float comparisons with zero may not work as expected. Please compare the original int value

Reply:
all right